scoreboard.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+
+    <title>Scoreboard | Serious game for kids</title>
+
+    <link rel="stylesheet" type="text/css" href="dist/scoreboard.css">
+    <script src="dist/scoreboard.js" type="application/javascript" async></script>
+</head>
+<body>
+    <nav class="navbar navbar-default">
+        <div class="container-fluid">
+            <div class="navbar-header">
+                <a class="navbar-brand" href="/scoreboard.html">Scoreboard</a>
+            </div>
+
+            <ul class="nav navbar-nav">
+                <li><a id="create-room" href="#">Create Room</a></li>
```

Comment:
(on multiple locations) these strings should be translation-ready, and ideally already be translated.

Reply:
Need to have a deeper look into this, since the translation part is only available in the main game. Any suggestions?